### DISCUSSION TITLE
copilot

### DISCUSSION THREAD

**Discussion by jonas-bl** [community/community#63430] [Copilot Conversations]
### Select Topic Area

Question

### Body

Is chatgpt the same as github copilot?

**Comment by ArvedLukas** [comment]
No chatgpt is different.

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
